org-formation/organization.yaml

```diff
@@ -619,6 +619,19 @@ Organization:
         budget-alarm-threshold: 2000
         budget-alarm-threshold-email-recipient: aws-openchallenges-prod@sagebase.org
 
+  SynapseLlmDevAccount:
```

Comment:
how about adding it to one of the OUs?  SynapseProdOU or ItProdOU perhaphs?

Reply:
Added to SynapseOU. This is for SCPs, correct?

Reply:
yes that is correct.